x/attribute/spec/01_state.md

```diff
@@ -0,0 +1,56 @@
+# State
+The attribute module inserts all attributes into a basic state store.
+
+# Attribute KV-Store
```

Comment:
This should be level 2 and the ones below it should be level 3.
Typically there's only ever one level 1 header in these files, and that's the one at the very top defining the whole document.

Reply:
Added